lib/cppcheck.cpp

```diff
@@ -1504,6 +1625,9 @@ void CppCheck::executeAddons(const std::vector<std::string>& files, const std::s
             }
             errmsg.file0 = file0;
 
+            setGuideline(errmsg);
```

Comment:
It's possible to adjust the guideline and classification here directly. It would also be possible to do it a bit later.
One advantage with doing it later would be that it avoids the serialization/deserialization problem.

However if we do it here then we should tweak the GUI.. it means the GUI doesn't need to determine the classification/guideline anymore.